include/sqlpp11/extra_tables.h

```diff
@@ -134,7 +146,7 @@ namespace sqlpp
       using _new_statement_t = new_statement_t<Check::value, Policies, no_extra_tables_t, T>;
 
       template <typename... T>
-      using _check = logic::all_t<is_table_t<T>::value...>;
+	  struct _check : logic::all_t<is_table_t<T>::value...>{};
```

Comment:
Could you run a recent clang-format with the configuration given in the project's root folder? See also the pre-commit sample.